third_party/packages/cupertino_icons/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.7
+
+* Adds example.md file to display usage of `cupertino_icons`
```

Comment:
This is missing a period; please see the style guide linked from the checklist.

Reply:
I have added a period at the end of the sentence. I will follow the style guide next time.